tests/test_dicom.py

```diff
@@ -117,6 +117,22 @@ def test_read_rgb(self, dicom_object):
         assert array.shape[-2:] == (100, 100)
         assert array.dtype == np.uint8
 
+    @pytest.mark.ci_skip  # CircleCI will not have a GPU
+    def test_nvjpeg(self, dicom_file_j2k: str):
+        ds = pydicom.dcmread(dicom_file_j2k)
+
+        start_time = time.time()
+        cpu_decompressed = read_dicom_image(ds, use_nvjpeg=False)
+        cpu_time = time.time() - start_time
+
+        start_time = time.time()
```

Comment:
I believe DICOM objects will cache `pixel_array` on first access/decode. Reopen the DICOM file to ensure that we're getting a fresh read.

```suggestion
        ds = pydicom.dcmread(dicom_file_j2k)
        start_time = time.time()
```